src/main/java/net/snowflake/client/jdbc/RestRequest.java

```diff
@@ -159,6 +159,7 @@ public static CloseableHttpResponse execute(
     // try request till we get a good response or retry timeout
     while (true) {
       logger.debug("Retry count: {}", retryCount);
+      logger.debug("Attempting request: {}", requestInfoScrubbed);
```

Comment:
In the future lets try to be explicit about what we're dumping in the logs - the current log statement leaves it to interpretation what the log actually is - especially for someone who might not be familiar with the code ( support team for ex ). Being explicit like `Attempting request with requestId: {}` would make it immediately clear what this is and also make it easier to do a word search.

Reply:
makes sense, thanks!